assets/styles/ui-components/header/_header-favorites.scss

```
@@ -0,0 +1,45 @@
+#js-favorites-open {
+    flex: 1 0 auto;
+    width: 2.375rem;
+    height: 2.375rem;
+
+    svg {
+        margin: 0 !important;
+    }
+
+    @include from($tablet) {
+        width: rem(50px);
+        height: rem(50px);
+    }
+}
+
+#js-favorites-container {
+    ul {
+        list-style: none;
+        padding: 0;
+
+        li {
+            &:not(:last-of-type) {
+                border-bottom: 2px solid $primary;
+            }
+
+            .button-container {
+                display: flex;
+                gap: 2rem;
```

```suggestion
                gap: $theme-spacing-two;
```

Olihan täällä noi spacing-muuttujat?